src/AutoHashEquals.jl

```diff
@@ -1,89 +1,46 @@
-VERSION < v"0.7.0-beta2.199" && __precompile__(true)
+# SPDX-License-Identifier: MIT
```

Comment:
what's this do 👀 ?

Reply:
Not a clue.  I don't know how it got there.

Reply:
should we delete?

Reply:
https://spdx.dev/ids/#:~:text=An%20SPDX%20ID%20is%20located,already%20present%20in%20your%20codebase.